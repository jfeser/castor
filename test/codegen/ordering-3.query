atuple([alist(orderby([f desc], r1) as k, atuple([ascalar(k.f), ascalar(k.g)], cross)), atuple([ascalar(9 as x), ascalar(9 as y)], cross), alist(orderby([f], r1) as k1, atuple([ascalar(k1.f), ascalar(k1.g)], cross))], concat)