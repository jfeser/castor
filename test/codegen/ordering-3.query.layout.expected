0:22 Tuple body
0:10 List body
0:2 Tuple body
0:1 Scalar (=(Int 3))
0:0 Tuple len (=22)
0:0 List len (=10)
0:0 List count (=5)
0:0 Tuple len (=2)
1:1 Scalar (=(Int 4))
2:2 Tuple body
2:1 Scalar (=(Int 2))
2:0 Tuple len (=2)
3:1 Scalar (=(Int 1))
4:2 Tuple body
4:1 Scalar (=(Int 2))
4:0 Tuple len (=2)
5:1 Scalar (=(Int 2))
6:2 Tuple body
6:1 Scalar (=(Int 1))
6:0 Tuple len (=2)
7:1 Scalar (=(Int 2))
8:2 Tuple body
8:1 Scalar (=(Int 1))
8:0 Tuple len (=2)
9:1 Scalar (=(Int 3))
10:2 Tuple body
10:1 Scalar (=(Int 9))
10:0 Tuple len (=2)
11:1 Scalar (=(Int 9))
12:10 List body
12:2 Tuple body
12:1 Scalar (=(Int 1))
12:0 List len (=10)
12:0 List count (=5)
12:0 Tuple len (=2)
13:1 Scalar (=(Int 2))
14:2 Tuple body
14:1 Scalar (=(Int 1))
14:0 Tuple len (=2)
15:1 Scalar (=(Int 3))
16:2 Tuple body
16:1 Scalar (=(Int 2))
16:0 Tuple len (=2)
17:1 Scalar (=(Int 1))
18:2 Tuple body
18:1 Scalar (=(Int 2))
18:0 Tuple len (=2)
19:1 Scalar (=(Int 2))
20:2 Tuple body
20:1 Scalar (=(Int 3))
20:0 Tuple len (=2)
21:1 Scalar (=(Int 4))
